apps/e2e/test/e2e_test.exs

```diff
@@ -354,7 +366,7 @@ defmodule E2ETest do
 
           [coordinates] = result["geometry"]["coordinates"]
 
-          assert 253 == Enum.count(coordinates)
```

Comment:
This was very flaky before and was probably only as flaky as before due to our change. Making it less flaky with this < Ben

Reply:
👍